src/App.tsx

```diff
@@ -1,61 +1,121 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { User } from './types/User';
```

Comment:
Try to fix all linter errors.
![image](https://github.com/mate-academy/react_add-todo-form/assets/56752562/fe18d5b1-baf1-44f0-bcea-35adb60c2632)